src/Commands/Remote/SSHBaseCommand.php

```diff
@@ -170,13 +171,44 @@ protected function validateFramework($framework)
     /**
      * Gets the command-line args
      *
-     * @param string $command_args
+     * @param string[] $command_args
      * @return string
      */
     private function getCommandLine($command_args)
     {
         array_unshift($command_args, $this->command);
 
-        return implode(" ", $command_args);
+        return implode(" ", $this->escapeArguments($command_args));
+    }
+
+    /**
+     * Escape the command-line args
+     *
+     * @param string[] $args All of the arguments to escape
+     * @param string[]
+     */
+    private function escapeArguments($args)
+    {
+        return array_map(
+            function ($arg) {
```

Comment:
Do we need to wrap this in another function? Shouldn't [$this, 'escapeArgument'] work? Or does it not because it's private?